fenix/nimbus.fml.yaml

```diff
@@ -138,6 +138,21 @@ features:
       - channel: nightly
         value:
           enabled: false
+  
+  client-deduplication:
+    description: A feature to control the sending of the client-deduplication ping.
+    variables:
+      enabled:
+        description: If true, the ping will be sent on a similar schedule as the baseline ping.
+        type: Boolean
+        default: false
+    defaults:
+      - channel: nightly
+        value:
+          enabled: true
+      - channel: developer
+        value:
+          enabled: true
```

Comment:
This will default to "true" for developer/debug local builds. I think you probably want this to be false so that you can test the same behavior as nightly, etc.